crates/rustfix/tests/parse_and_replace.rs

```diff
@@ -238,10 +238,3 @@ fn everything() {
     tracing_subscriber::fmt::init();
     assert_fixtures("./tests/everything", fixmode::EVERYTHING);
 }
-
-#[test]
-#[ignore = "Requires custom rustc build"]
-fn edition() {
```

Comment:
Guess you might want to remove all references to `fixmode::EDITION`, and the `./test/edition` directory as well.

https://github.com/rust-lang/cargo/blob/35ed69cb1b32fcb66c77032cc5480965fb519558/crates/rustfix/tests/parse_and_replace.rs#L16

Reply:
Doing so leaves a useless param to calls to compile.  Will unwind that next unless you see a reason to keep.